ChooseArea.svelte

```diff
@@ -103,9 +105,16 @@
   <br />
 
   <p>Select Transport Authority or Local Authority District:</p>
-  <input list="authorities-list" bind:this={inputList} />
-  <datalist id="authorities-list" bind:this={dataList} />
-  <button type="button" on:click={start}>Start</button>
+  <div>
+    <input list="authorities-list" bind:value={inputValue} />
```

Comment:
I realized it was easier to just `bind:value` instead of binding a DOM element and then doing `x.value`